src/docs/asciidoc/resources/migration_guide_to_4.6.adoc

```diff
@@ -67,4 +67,8 @@ The new default configuration is as such :
   dayOfWeekToSendWeeklyRecapEmail: 1
   hourToSendRecapEmail: 7
   minuteToSendRecapEmail: 30
-```
\ No newline at end of file
+```
+
+== Timezone
+
+For the cards-external-diffusion service, the timezone configuration has been moved from the Dockerfile to the docker-compose.yml file. This modification allows to have your local timezone as the reference for the service. If nothing is set in Dockerfile, the default value will be the local timezone.
```

Comment:
If nothing is set in the  docker compose file , the default value will be the machine local timezone